app/assets/javascripts/discourse/app/components/user-tip.hbs

```diff
@@ -1 +1 @@
-<span {{did-insert this.showUserTip}}></span>
\ No newline at end of file
+<span class="user-tip-span" {{did-insert this.showUserTip}}></span>
```

Comment:
Generally I would try to give a more meaningful name, and if you can't find anything, container is probably better suited